File: src/main/java/pe/nanamochi/poc_osu_server/packets/client/StatusUpdateRequestPacket.java
package pe.nanamochi.poc_osu_server.packets.client;

import lombok.Data;
import lombok.NoArgsConstructor;
import pe.nanamochi.poc_osu_server.packets.Packet;
import pe.nanamochi.poc_osu_server.packets.Packets;

@Data
@NoArgsConstructor
public class StatusUpdateRequestPacket implements Packet {
    @Override
    public Packets getPacketType() {
        return Packets.OSU_STATUS_UPDATE_REQUEST;
    }
}

File: banchus-packet/src/main/kotlin/pe/nanamochi/banchus/packets/client/StatusUpdateRequestPacket.kt
package pe.nanamochi.banchus.packets.client

import java.io.InputStream
import pe.nanamochi.banchus.core.BanchoPacket
import pe.nanamochi.banchus.core.PacketType
import pe.nanamochi.banchus.io.DataReader

class StatusUpdateRequestPacket : BanchoPacket(), BanchoPacket.Client {
    override val type = PacketType.OSU_STATUS_UPDATE_REQUEST

    override fun read(reader: DataReader, ins: InputStream) {}
}
